server/controllers/createPosts.js

```diff
@@ -23,17 +22,25 @@ const Post = (req, res) => {
   } = req.body;
 
   if (req.body.price <= 1000) {
-return res.status(400).json({status: 400, message:"Not allowed to post, the price is low" }); 
-  }
 
-  const newPost = {
-    id, email, status:("Available") , type, state, city, address, price,
+    return res.status(400).json({ status: 400, message: "Not allowed to post, the price is low" });
```

Comment:
The key to status codes that are not 201 0r 200 should be `error` not `message`.